### ISSUE TITLE
Help with BIOMOD- low TSS and Kappa scores

### ISSUE THREAD

**Issue by ronya779** [biomodhub/biomod2#675]
Hi BIOMOD2 Team,
I don't have much experience with the Biomod2 package yet. I am currently having trouble achieving good TSS and Kappa values for my modeling. 
The plant I am looking at is an invasive species and has a relatively large ecological niche. It is a presence (113 points) with true absence dataset (680 points). I have selected 9 dependent variables to have at least 10 occurrences per variable. I am currently using the RF, BRT, MARS, and GLM algorithms. Ultimately, I don't want to create a model ensemble, but rather find out which variables have the greatest influence on the occurrence of the plant and which model is most suitable. 

The average values for all models are AUCroc 0.78, TSS 0.344, Kappa 0.26, Accuracy 0.84. When I use the “bigboss” setting, the values are AUCroc 0.82, TSS 0.42, Kappa 0.35, Accuracy 0.85. When I use bm_Tuning, the values are similar to the bigboss settings. 

The calibration value for RF is always 1, while the average calibration values are 0.18 (TSS), 0.80 (AUCroc), 0.23 (Kappa), and 0.85 (Accuracy). 
For the other 3 algorithms, the calibration values are always close to the validation values. However, TSS (0.4) and Kappa (0.3) are relatively poor there as well.
After using bm_tuning, the calibration value was 1 for all four algorithms. However, I then had the problem with RF that my sample size was too large, so I manually set the value to zero. 
I also tried working with spatial blocking, but this led to the values deteriorating further.
I have also already reduced the variables. Are there any other possibilities for obtaining good TSS and Kappa values that I have not mentioned? 

my_options <- bm_ModelingOptions(
  data.type = "binary",
  models = c("GLM", "MARS", "RF", "GBM"),
  strategy = "default", (or "bigboss")
  bm.format = my_data_n,)

OR

tuned_rf <- bm_Tuning(model = "RF",
                      tuning.fun = "rf", ## see in ModelsTable
                      do.formula = TRUE,
                      bm.options = my_options@options$RFd.binary.randomForest.randomForest,
                      metric.eval = "TSS",
                      bm.format = my_data_n)

tuned_gam <- bm_Tuning(model = "GBM",
                       tuning.fun = "gbm", ## see in ModelsTable
                       do.formula = TRUE,
                       do.stepAIC = TRUE,
                       bm.options = my_options@options$GBM.binary.gbm.gbm,
                       metric.eval = "TSS",
                       bm.format = my_data_n)

my_options_tuning <- bm_ModelingOptions(
                        data.type = "binary",
                        models = c("RF", "GBM", "MARS", "GLM"), #"MARS", "GLM",
                        strategy = "user.defined",
                        user.val = list(GBM.binary.gbm.gbm = tuned_gam,
                                        RF.binary.randomForest.randomForest = tuned_rf),
                        bm.format = my_data_n)
  
mymodel<-BIOMOD_Modeling(my_data_n, models = c("RF", "GBM", "GLM", "MARS"),
                         CV.strategy = "kfold",
                         CV.do.full.models = TRUE,
                         CV.nb.rep = 5,
                         CV.k = 5,        
                         OPT.user = my_options, (or my_options_tuning )
                         var.import = 3,
                         metric.eval = c("TSS","ROC", "KAPPA", "ACCURACY"),
                         do.progress = TRUE,
                         modeling.id = "121",

**Comment by MayaGueguen** [comment]
Hello there 👋 

Thank you for trying biomod2 !

Would you mind sending me the outputs of the following commands so I can get a better idea of your data ? 👀 
```
print(my_data_n)
summary(my_data_n)
plot(my_data_n)
```

Maya

**Comment by MayaGueguen** [comment]
Thank you for the informations 🙏 
Coud I get the distribution plot as well ?
